src/20_Components/amp-date-picker.html

```diff
@@ -0,0 +1,202 @@
+    The `blocked` attribute allows to specify a space separated list of ISO 8601 dates and RFC 5545 RRULEs specifying disallowed dates.
+    The `highlighted` attributes a space separated list of ISO 8601 dates and RFC 5545 RRULEs specifying dates displayed with a highlight style.
+  -->
+  <div>
+    <button class="ampstart-btn m1 caps" on="tap:simple-date-picker.clear">Clear</button>
+    <amp-date-picker
+      type="single"
+      layout="container"
+      width="300"
+      height="300"
+      on="select:AMP.setState({date: event.date, dateType: event.id})"
+      locale="en"
+      format="Y-MM-DD"
+      number-of-months="1"
+      show-clear-date
+      keep-open-on-date-select
+      highlighted="2017-10-10 2017-10-12"
+      blocked="2017-10-10 2017-10-17 2017-10-19 2017-10-21"
+      id="simple-date-picker-3"
+      class="mr1 ml1">
+      <input amp-date-placeholder placeholder="Pick a date!!"/>
+      <template type="amp-mustache" info-template>
+        <span [text]="date != null ? 'You picked ' + date + '.' : 'You will see your chosen date here.'">You will see your chosen date here.</span>
+      </template>
+    </amp-date-picker>
+  </div>
+  <!-- #### Date Range Picker -->
+  <!--
+    `amp-date-picker` with `type=range` can be used for selecting a date range. The default css will disable
+    past dates, use `min` attribute to enable them.
+  -->
+  <div>
+  <button class="ampstart-btn m1 caps" on="tap:range-date-picker.setDates(startDate='2017-10-26',endDate='2017-11-04')">Set date range</button>
+  <button class="ampstart-btn m1 caps" on="tap:range-date-picker.clear">Clear</button>
+  <amp-date-picker
+    type="range"
+    id="range-date-picker"
+    on="
+    select:
+      AMP.setState({
+          dates: event.dates,
+          startDate: event.dates[0].date,
+          endDate: event.dates[event.dates.length-1].date
+      })"
+    locale="en"
+    format="Y-MM-DD"
+    keep-open-on-date-select
+    show-clear-dates
+    min="2017-10-26"
+    class="mr1 ml1">
+    <input amp-date-placeholder-start placeholder="Start date" id="start" />
+    <input amp-date-placeholder-end placeholder="End date" id="end" />
+    <template type="amp-mustache" info-template>
+      <span [text]="'You picked ' + startDate + ' as start date and ' + endDate + ' as end date.'">You will see your chosen dates here.</span>
+    </template>
+  </amp-date-picker>
+
+
+  </div>
+
+  <!-- #### External configuration -->
+  <!--
+    It's possible to configure the `amp-date-picker` preferences by using an external json. We are using the following:
+```
+    {
+      "templates": [{
+      "id": "low",
+      "dates": [
+        "2017-11-01", "2017-11-03", "2017-11-05", "2017-11-06",
+        "FREQ=WEEKLY;DTSTART=20171101T150000Z;COUNT=30;WKST=MO;BYDAY=TU,SA"
+        ]
+      }]
+    }
+```
+  and we set the `src` of amp-date-picker with the filename amp-date-picker.json -->
```

Comment:
* Can you mark amp-date-picker and amp-date-picker.json with "`"
* s/and/Then

Reply:
done